make-macosx-app.sh

```diff
@@ -336,6 +349,11 @@ if [ -n "${MACOSX_DEPLOYMENT_TARGET_PPC}" ] || [ -n "${MACOSX_DEPLOYMENT_TARGET_
         <key>x86_64</key>
         <string>${MACOSX_DEPLOYMENT_TARGET_X86_64}</string>"
 	fi
+	if [ -n "${MACOSX_DEPLOYMENT_TARGET_ARM64}" ]; then
+	PLIST="${PLIST}
+        <key>x86_64</key>
```

Comment:
Key should be arm64 or something, right?

Reply:
Probably. The person I had test it used the make-macosx.sh script so it didn't come up. I'll add it.